centaur/src/main/scala/centaur/test/metadata/WorkflowFlatMetadata.scala

```diff
@@ -79,17 +81,62 @@ object WorkflowFlatMetadata {
     }
   }
 
-  def fromMetadataJson(json: WorkflowMetadata): ErrorOr[WorkflowFlatMetadata] = {
-    import DefaultJsonProtocol._
-    Try(json.value.parseJson.asJsObject.flatten().convertTo[Map[String, JsValue]]) match {
-      case Success(m) => Valid(WorkflowFlatMetadata(m))
-      case Failure(e) => invalidNel(s"Unable to create Metadata from JSON: ${e.getMessage}")
+  def fromWorkflowMetadata(workflowMetadata: WorkflowMetadata): ErrorOr[WorkflowFlatMetadata] = {
+    val jsValue: ErrorOr[JsValue] = Try(workflowMetadata.value.parseJson) match {
+      case Success(jv) => Valid(jv)
+      case Failure(e) => invalidNel(s"Unable to create JsValue from String: ${e.getMessage}")
+    }
+
+    for {
+      jv <- jsValue
+      map <- jv.asMap
+    } yield WorkflowFlatMetadata(map)
+  }
+
+  implicit class EnhancedWorkflowMetadata(val workflowMetadata: WorkflowMetadata) {
+    def asFlat: WorkflowFlatMetadata = {
+      WorkflowFlatMetadata.fromWorkflowMetadata(workflowMetadata).unsafe
+    }
+  }
+
+  implicit class EnhancedWorkflowFlatMetadata(val workflowFlatMetadata: WorkflowFlatMetadata) {
+    def stringifyValues: Map[String, JsValue] = {
+      import mouse.all._
+      workflowFlatMetadata.value.map {
+        case (k, v: JsString) => k -> v
+        case (k, JsNull) => k -> JsNull
+        case (k, v) => k -> (v.toString |> JsString.apply)
+      }
+    }
+  }
+}
+
+object WorkflowFlatOutputs {
+  implicit class EnhancedWorkflowOutputs(val workflowOutputs: WorkflowOutputs) extends AnyVal {
+    def asFlat: WorkflowFlatMetadata = {
+      workflowOutputs.outputs.asMap map WorkflowFlatMetadata.apply unsafe
```

Comment:
Is the postfix ops import just so you can do this without the `.`? 

TOL I guess, but IMO I'd rather just say `[...].apply.unsafe` and not import the postfixops